src/components/navBar/navBar.js

```diff
@@ -130,6 +130,24 @@ function MdNavBar($mdAria, $mdTheming) {
         '<md-nav-ink-bar ng-hide="ctrl.mdNoInkBar"></md-nav-ink-bar>' +
       '</div>',
     link: function(scope, element, attrs, ctrl) {
+
+      scope.width = $window.innerWidth;
```

Comment:
Can this be a variable here in the function instead of on the `scope`?

Reply:
Tried with `ctrl` seems to be working fine.